modules/3d/src/rgbd/odometry_frame_impl.cpp

```diff
@@ -0,0 +1,253 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html
+
+#include "../precomp.hpp"
+
+#include <iostream>
+#include <opencv2/core.hpp>
+#include <opencv2/core/ocl.hpp>
+#include <opencv2/core/mat.hpp>
```

Comment:
> iostream

Avoid using of that header (I left detailed comment about that on this month - check project conversations in email).
Use logging.

> core.hpp
> mat.hpp

Not needed, included by default.

Reply:
done